admin/src/collections/Users.ts

```diff
@@ -89,7 +89,7 @@ export const usersCollection = buildCollection<User>({
 	}),
 	additionalColumns: [FirstNameCol, LastNameCol, GenderCol, PhoneCol, CountryCol, CityCol, ReferralCol],
 	subcollections: [contributionsCollection, donationCertificateCollection],
-	extraActions: CreateDonationCertificatesAction,
+	Actions: CreateDonationCertificatesAction,
```

Comment:
what's this change about?

Reply:
Sandino wished to upgrade FireCMS, which did in this PR as well. The parameter name just changed, so I updated it.